dissect/target/helpers/configutil.py

```diff
@@ -180,6 +180,43 @@ def read_file(self, fh: TextIO | io.BytesIO) -> None:
         if not isinstance(self.parsed_data, dict):
             self.parsed_data = self._collapse_dict(self.parsed_data, False)
 
+    def merge(self, other: ConfigurationParser) -> ConfigurationParser:
```

Comment:
Reviewer: could push this down to SystemDParser

Reply:
I think it's fine, could maybe be useful for other parsers in general.